src/__experimental__/components/label/label.component.js

```diff
@@ -109,12 +111,15 @@ Label.propTypes = {
   /** Set focus possibilities to an <IconWrapperStyle /> element.
    *  More information: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
   */
-  tabIndex: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+  tabIndex: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  /** Allows to override existing component styles */
+  styleOverride: PropTypes.oneOfType([PropTypes.func, PropTypes.object])
```

Comment:
Why does this include `PropTypes.func`?  It looks like only an object would work


```js
${({ styleOverride }) => styleOverride};
```

Reply:
when you pass a function, then you'll have access to styled component props in that function arguments (theme object for example). The function is automatically invoked, It's counterintuitive, but that is how it works in Styled Components.

Reply:
Thanks for clarifying 🙂